doc/blank-raw-image.md

```diff
@@ -0,0 +1,69 @@
+# How to create Blank Raw Image User Guide
+The purpose of this document is to show how to create a new blank raw image in your PersistentVolumeClaim resource.
+
+## Prerequesites
+You have a Kubernetes cluster up and running with CDI installed and at least one PersistentVolume is available.
+
+## Create Blank Raw Image with DataVolume manifest
+
+Create the following DataVolume manifest (blank-image-datavolume.yaml):
+
+```bash
+apiVersion: cdi.kubevirt.io/v1alpha1
+kind: DataVolume
+metadata:
+  name: blank-image-datavolume
+spec:
+  source:
+      blank: {}
+  pvc:
+    #the storageClassName is optional, if missing we use the default storage class.
```

Comment:
Maybe better to keep using the convention in line 54. I.e. "# Optional: ..."

Reply:
Both DataVolume an PVC were taken from manifest/example.